plat/common/pci_ecam.c

```diff
@@ -226,14 +226,14 @@ int gen_pci_irq_parse(const fdt32_t *addr, struct fdt_phandle_args *out_irq)
 	fdt32_t initial_match_array[16];
 	const fdt32_t *match_array = initial_match_array;
 	const fdt32_t *tmp, *imap, *imask;
-	const fdt32_t dummy_imask[] = { [0 ... 16] = cpu_to_fdt32(~0) };
-	int intsize, newintsize;
```

Comment:
Is there a reason this line is removed? Has this patch been tested? 🧐

Reply:
Nope. My mistake, apologies for that.